clippy_lints/src/utils/conf.rs

```diff
@@ -106,6 +106,8 @@ macro_rules! define_Conf {
 
 pub use self::helpers::Conf;
 define_Conf! {
+    /// Lints: MANUAL_NON_EXHAUSTIVE, MANUAL_STRIP, OPTION_AS_REF_DEREF, MATCH_LIKE_MATCHES_MACRO. The minimum rust version that the project supports
```

Comment:
```suggestion
    /// Lints: MANUAL_NON_EXHAUSTIVE, MANUAL_STRIP, OPTION_AS_REF_DEREF,
    /// MATCH_LIKE_MATCHES_MACRO. The minimum rust version that the project
    /// supports
```
Readability suggestion

Reply:
I don't think we can do this, since our scripts won't pick up the correct configurations anymore. We would have to test this though.

Reply:
I just tested it, and no we can't do this. We could improve the situation here, but that's not for this PR to do.